chore: add new Prometheus endpoint information (#1318)
falcosecurity/falco-website: content/en/docs/metrics/falco-metrics.md

```
  
</details>

> ## Monitoring

Reviewer: Monitoring is too generic and seems out of context having it here as we already talk about metrics above.

> ## Monitoring

Author: Good for me, we can always change it in the future if we provide other outputs.

> ## Monitoring

Reviewer: Agree we can come up with a better doc once we cross that bridge.

> Since version 0.38.0, you can use Falco's web server to monitor its state. A new `/metrics` endpoint has been added which provides metrics information that can be collected by [Prometheus][2].

Reviewer: ```suggestion
Since version 0.38.0, you can also use Falco's web server to expose Falco's metrics. A new `/metrics` endpoint has been added which provides metrics information that can be collected by [Prometheus][2].
```
